plist

```diff
@@ -1087,17 +1087,15 @@
 /usr/local/opnsense/scripts/captiveportal/strip_template.py
 /usr/local/opnsense/scripts/dhcp/cleanup_leases4.php
 /usr/local/opnsense/scripts/dhcp/cleanup_leases6.php
-/usr/local/opnsense/scripts/dhcp/dnsmasq_watcher.py
```

Comment:
Unsure about this one. It’s only needed when isc is used so theoretically it can move to plugin.